microsoft-edge/devtools-guide-chromium/storage/cookies.md

```diff
@@ -5,7 +5,7 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 05/04/2021
+ms.date: 06/08/2022
```

Comment:
```suggestion
ms.date: 09/08/2022
```

Reply:
@mikehoffms you probably noticed, there are two new fields in the cookies table that are blank because I don't know what/why they are.

Reply:
@dan-wesley Pls close this Outdated conversation if resolved.

Reply:
@dan-wesley (or @captainbrosset )  Please close this "Outdated" conversation as Resolved if this has been resolved in the latest files.